x-pack/plugins/monitoring/public/application/contexts/header_action_menu_context.tsx

```diff
@@ -14,3 +14,15 @@ interface ContextProps {
 }
 
 export const HeaderActionMenuContext = React.createContext<ContextProps>({});
+
+export const HeaderActionMenuProvider: React.FC<Required<ContextProps>> = ({
```

Comment:
Where does this get used? Seems like I can remove it and still get a drop down.

Reply:
Good catch!  I meant to replace [this](https://github.com/elastic/kibana/blob/main/x-pack/plugins/monitoring/public/application/index.tsx#L105) with this wrapper but I'm just going to remove this and let SM call it explicitly like it does with other providers.